Fix typo in rgba() syntax

pallxk opened the pull request:
<!-- Please provide the following information to help us review this PR: -->

> What was wrong/why is this fix needed? (quick summary only)

`rgba(255 255 255,.5);` is invalid, it should be `rgba(255 255 255/.5);` or styled in other ways like `rgba(255 255 255 / .5);`

> MDN URL of main page changed

https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/rgba()

> Issue number (if there is an associated issue)

NA

> Anything else that could help us review it

* Spec: https://www.w3.org/TR/css-color-4/#rgb-functions
* Related MDN page: https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/rgb()

pallxk committed:
Fix typo in rgba() syntax

https://www.w3.org/TR/css-color-4/#rgb-functions

ericwbailey commented on files/en-us/web/css/color_value/rgba()/index.html:
```diff
@@ -22,7 +22,7 @@ <h4>Note</h4>
 
 <pre class="brush: css">rgba(255,255,255,.5) /* white with 50% opacity */
-rgba(255 255 255,.5); /* CSS Colors 4 space-separated values */
+rgba(255 255 255/.5); /* CSS Colors 4 space-separated values */
```
```suggestion
rgba(255 255 255 / 0.5); /* CSS Colors 4 space-separated values */
```

I'd like to see spaces between the slash and a leading zero here.

ericwbailey commented:
Nice catch, @pallxk, thank you! One suggestion and then I think this is good to merge 🎉

pallxk committed:
Style arguments of rgba()

Add spaces between parameters and slash separator;
Add a leading zero to fractional numbers.

Co-authored-by: Eric Bailey <ericwbailey@users.noreply.github.com>

pallxk commented:
Fixed! 😃

ericwbailey reviewed (approved):
Woo! Approved and merged! Congrats on your first contribution 🎉
